host_classifer.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# The input inventory file(s)
+FILES=$@
+
+# Inventory we'll increment as we find systems
```

Comment:
you are accumulating counts for all files as a whole, rather than each file individually